diff --git a/node_modules/fastify-method-override/lib/index.js b/node_modules/fastify-method-override/lib/index.js
index df082b9..0b97696 100644
--- a/node_modules/fastify-method-override/lib/index.js
+++ b/node_modules/fastify-method-override/lib/index.js
@@ -19,7 +19,7 @@ const getHooks = (routeOptions, hookName) => {
 };
 const hooksTable = ['preValidation', 'preHandler'];
 const getAllHooks = routeOptions => _lodash.default.flatMap(hooksTable, hookName => getHooks(routeOptions, hookName));
-const fastifyMethodOverride = async (fastify, opts, next) => {
+const fastifyMethodOverride = async (fastify, opts) => {
   const allowMethods = new Set(['head', 'put', 'delete', 'options', 'patch']);
   const routeMatchers = {};
   const handleRedirect = async (req, reply) => {
@@ -108,7 +108,6 @@ const fastifyMethodOverride = async (fastify, opts, next) => {
       await handleRedirect(req, reply);
     }
   });
-  next();
 };
 var _default = (0, _fastifyPlugin.default)(fastifyMethodOverride, {
   fastify: '>=2.0.0',
